agent/install_into

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
Can this just be moved into the makefile target? I.e

```
cd build; npm install --production
```

The cd will only affect the shell invocation on that line.

Reply:
Yes, that is much nicer! Done now.